torch/nn/_functions/dropout.py

```diff
@@ -59,9 +59,11 @@ class FeatureDropout(Dropout):
 
     @staticmethod
     def symbolic(g, input, p=0.5, train=False, inplace=False):
-        # NB: No FeatureDropout in ONNX, use Dropout instead.
+        # NB: In inference mode, FeatureDropout is exported as an identity op.
         if train:
-            raise ValueError("In train mode, FeatureDropout is different from Dropout.")
+            raise ValueError("Exporting ONNX FeatureDropout in train mode is not supported, "
+                             "and the exported FeatureDropout should not be used for training. "
```

Comment:
I'm not sure what this sentence means. Isn't it saying the same thing as the first sentence?